### PR TITLE
Make sure dashel properly exposes its include directory

### PR DESCRIPTION
**Pull request by cor3ntin**

### CODE REVIEW

**Comment by stephanemagnenat** [comment]
Just to make sure, does this work with the minimum CMake required of Dashel (2.6)?

**Comment by stephanemagnenat** [comment]
Another question: why is this change in `CMakeLists.txt` and not in `dashelConfig.cmake.in`?

File: CMakeLists.txt [hunk 1]
```diff
@@ -52,6 +52,12 @@ endif (WIN32)
 add_library(dashel ${DASHEL_SOURCES})
 target_link_libraries(dashel ${EXTRA_LIBS})
 
+target_include_directories(dashel PUBLIC
+$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+$<INSTALL_INTERFACE:>
+
```
Comment (stephanemagnenat) [inline CMakeLists.txt]: strange spacing...

**Comment by davidjsherman** [comment]
Re: https://github.com/aseba-community/dashel/pull/41#issuecomment-364012127 a matrix of old and new build environments could be containerized to test this kind of CMake change. Not CI but specific to the appropriate feature branch

**Comment by cor3ntin** [comment]
@stephanemagnenat No. i'll put a check around.
It's not in dashelConfig.cmake because it helps using dashel as a directory ( via `add_subdirectory`) rather than a package.

**Comment by stephanemagnenat** [comment]
Ok I see. Could you please fix the extra spacing and then I merge.

**Pull request merged**